src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
be more attentive, you've missed the point from the checklist 
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop